Projects/Team3/Mosiychuk Tania/FirstSeleniumTest/UnitTest1.cs

```diff
@@ -0,0 +1,46 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System.Threading;
+using System;
+
+namespace FirstSeleniumTest
+{
+    public class Test
+    {
+        private IWebDriver driver;
+
+        [OneTimeSetUp]
+        public void BeforeAllMethods()
+        {
+            driver = new ChromeDriver();
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+        }
+
+        [OneTimeTearDown]
+        public void AfterAllMethods()
+        {
+            driver.Quit();
+        }
+
+        [Test]
+        public void FirstTest()
+        {
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            driver.Navigate().GoToUrl("https://ilearn.org.ua/login");
+            Thread.Sleep(2000);
```

Comment:
try to avoid Thread.Sleep
it is better just to increase ImplicitWait

Reply:
Okey, thank you